#!/usr/bin/env python
PACKAGE = "puthand_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("serial_device", str_t, 0, "Name of the serial device", "/dev/ttyUSB0")
gen.add("finger_reset_speed", double_t,0, "Speed of fingers during reset, given as percentage", 0.2,0.0,1.0)
gen.add("reset_timeout",int_t,0,"Time in seconds after which a reset will be terminated if fingers are stuck",5,0,60)

exit(gen.generate(PACKAGE, "puthand_controller", "puthand"))

